src/ray/core_worker/task_interface.h

```diff
@@ -0,0 +1,96 @@
+#ifndef RAY_CORE_WORKER_TASK_INTERFACE_H
+#define RAY_CORE_WORKER_TASK_INTERFACE_H
+
+#include "common.h"
+#include "ray/common/buffer.h"
+#include "ray/id.h"
+#include "ray/status.h"
+
+namespace ray {
+
+class CoreWorker;
+
+/// Options of a task call.
+struct CallOptions {
```

Comment:
need a constructor to initialize these const fields, since they cannot be changed directly

Reply:
I think we can use list initialization, e.g., `CallOptions call_options{1, {}}`?
It's a bit verbose to write constructor for every these POD (plain old data) type.